src/scroll-timeline-base.js

```diff
@@ -669,7 +669,8 @@ export function calculateRange(phase, sourceMeasurements, subjectMeasurements, a
 
   let startOffset = undefined;
   let endOffset = undefined;
-  const targetIsTallerThanContainer = viewSize > sizes.containerSize ? true : false;
+  const adjustedScrollportSize = sizes.containerSize - inset.start - inset.end;
```

Comment:
Might be good to add a comment here to why this is done:

```suggestion
  // Take inset into account when determining the scrollport size
  const adjustedScrollportSize = sizes.containerSize - inset.start - inset.end;
```